cuqi/experimental/mcmc/_rto.py

```diff
@@ -36,18 +36,24 @@ class LinearRTO(Sampler):
     tol : float
         Tolerance of the inner CGLS solver. *Optional*.
 
+    independent : bool
+        This will influence how the solver is set up in step().
+        If True, the initial point for the inner CGLS solver is set to the MAP estimate of the unperturbed linear least squares problem, and the samples will be independent.
+        If False, the initial point for the inner CGLS solver is set to the current point of the sampler, and the samples will be dependent.
+
     callback : callable, optional
         A function that will be called after each sampling step. It can be useful for monitoring the sampler during sampling.
         The function should take three arguments: the sampler object, the index of the current sampling step, the total number of requested samples. The last two arguments are integers. An example of the callback function signature is: `callback(sampler, sample_index, num_of_samples)`.
         
     """
-    def __init__(self, target=None, initial_point=None, maxit=10, tol=1e-6, **kwargs):
+    def __init__(self, target=None, initial_point=None, maxit=10, tol=1e-6, independent=False, **kwargs):
```

Comment:
I suggest renaming independent to something more explicit. For example: "inner_initial_point". This also works well with the naming convention in RegularizedRTO. For example, we have `maxit` and `inner_max_it`, `abstol` and `inner_abstol`. Then we will have `initial_point` and `inner_initial_point`.
The values of `inner_initial_point` can then be "map" or "previous_sample", for example.